### ISSUE TITLE
could LTRpred detect DIRS/DIRS in genome contigs?

### ISSUE THREAD

**Issue by wangj2009** [HajkD/LTRpred#9]
![image](https://user-images.githubusercontent.com/37939028/66776727-490a0d00-ee84-11e9-8a42-13221936cb82.png)

0. Could LTRpred detect DIRS/DIRS?
1.  Does LTRpred care about the LTR direction?
2. Does LTRpred care about the length of TSD? Could I set the parameter as mindistltr = 1000?
3. LTRpred use the default parameter overlaps = "all", but could I set the parameter as overlaps = "best"?
4. Are there any other parameters need to be set?
5. Are there some softwares to annotate DIRS/DIRS except LTRpred?
Thanks,

**Comment by HajkD** [comment]
Could LTRpred detect DIRS/DIRS?

> It won't detect inverted LTRs, but for same directional LTRs you could specify the correct protein domains that characterize for DIRS and screen for them with LTRpred

Does LTRpred care about the LTR direction?

> yes

Does LTRpred care about the length of TSD? Could I set the parameter as mindistltr = 1000?

> yes, see parameters `min_tsd`, `max_tsd`

LTRpred use the default parameter overlaps = "all", but could I set the parameter as overlaps = "best"?

> yes, see the documentation in R `?LTRpred::LTRpred`

Are there any other parameters need to be set?

https://hajkd.github.io/LTRpred/articles/Introduction.html

Are there some softwares to annotate DIRS/DIRS except LTRpred?

>I am sure there are.
